[2024-07-13T19:20:37.140+0100] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-13T19:20:37.370+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hhh.get_sentiments manual__2024-07-13T18:18:21.985875+00:00 [queued]>
[2024-07-13T19:20:37.379+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hhh.get_sentiments manual__2024-07-13T18:18:21.985875+00:00 [queued]>
[2024-07-13T19:20:37.379+0100] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-13T19:20:37.392+0100] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): get_sentiments> on 2024-07-13 18:18:21.985875+00:00
[2024-07-13T19:20:37.411+0100] {logging_mixin.py:188} WARNING - /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=9140) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-13T19:20:37.412+0100] {standard_task_runner.py:63} INFO - Started process 9153 to run task
[2024-07-13T19:20:37.413+0100] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'hhh', 'get_sentiments', 'manual__2024-07-13T18:18:21.985875+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/dg1.py', '--cfg-path', '/var/folders/ml/zd6cs33j1gs1prd5lpns9pnc0000gn/T/tmpe10tly6p']
[2024-07-13T19:20:37.415+0100] {standard_task_runner.py:91} INFO - Job 36: Subtask get_sentiments
[2024-07-13T19:20:37.482+0100] {task_command.py:426} INFO - Running <TaskInstance: hhh.get_sentiments manual__2024-07-13T18:18:21.985875+00:00 [running]> on host MAKs-MacBook-Pro.local
[2024-07-13T19:20:37.831+0100] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='hhh' AIRFLOW_CTX_TASK_ID='get_sentiments' AIRFLOW_CTX_EXECUTION_DATE='2024-07-13T18:18:21.985875+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-13T18:18:21.985875+00:00'
[2024-07-13T19:20:37.861+0100] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-13T19:20:43.294+0100] {python.py:237} INFO - Done. Returned value was:            ds  negative  positive  ...   sadness  surprise    others
0  2024-07-13  0.004165  0.317974  ...  0.000974  0.002793  0.980527

[1 rows x 11 columns]
[2024-07-13T19:20:43.318+0100] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-13T19:20:43.343+0100] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hhh, task_id=get_sentiments, run_id=manual__2024-07-13T18:18:21.985875+00:00, execution_date=20240713T181821, start_date=20240713T182037, end_date=20240713T182043
[2024-07-13T19:20:43.432+0100] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-13T19:20:43.455+0100] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-13T19:20:43.458+0100] {local_task_job_runner.py:222} INFO - ::endgroup::
